Assets/_FD/Script/AutoDestroy.cs

```diff
@@ -18,10 +18,12 @@ public void Init(float delay)
     IEnumerator DisableCo()
     {
         yield return null;
-
+        ScreenShake.instance.StartShake(0.2f, 0.1f);
```

Comment:
Can you add this to the Boss/MiniBoss walking too ! Its very good
like when a boss/miniboss walks add a very small amount of shake.
Good job 👍

Reply:
Yes, I will definitely add this feature and send it to you again tomorrow.